Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ghcr.io/linuxserver/baseimage-alpine-nginx:3.13
+
+ARG BUILD_DATE
+ARG VERSION
+ARG GRAV_RELEASE
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="TheSpad"
+
+RUN \
+  apk add --update --no-cache curl composer php7-dom php7-gd php7-tokenizer php7-opcache php7-pecl-apcu php7-pecl-yaml unzip && \
```

Comment:
This would be easier to read on multiple lines. ex: https://github.com/linuxserver/docker-swag/blob/36495ea20464ae587a53f08371e82d34aa4ccc91/Dockerfile#L16-L22